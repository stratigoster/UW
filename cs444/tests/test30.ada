package foo is
  type boo is
    record
      f : float; -- universal float
    end record;
  type float is -- redefining float
    record
      null;
    end record;
  b : boo;
  i : float := b.f;
--             ^^^
-- incompatible assignment of `universal float' to `float'
-- since `float' is now a record
end foo;
