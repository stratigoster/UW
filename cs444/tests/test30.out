float float
***ERROR***:11:5:	Incompatible types in assignment of ` universal float ' to ` float '

------
Compilation error
------
